crates/audio/Cargo.toml

```diff
@@ -16,7 +16,7 @@ collections = { path = "../collections" }
 util = { path = "../util" }
 
 
-rodio ={version = "0.17.1", default-features=false, features = ["wav"]}
```

Comment:
Hmm, I believe we need that feature as the sounds from `assets` are all in .wav format.

Reply:
Yeah, I've just checked and this would break notification sounds in Zed.

Reply:
Thanks for confirming. I think the rodio crate should change how they use features, but regardless, I should have done my due diligence on this one. I've added it back.